src/commands/runtime/action/create.js

```diff
@@ -172,7 +172,7 @@ class ActionCreate extends RuntimeBaseCommand {
         limits = limits || {}
         limits.timeout = flags.timeout
       }
-      if (flags.logsize) {
```

Comment:
That is why I don't like weekly typed languages... We're checking for boolean where `flags.logsize` is actually an integer. 
I would argue that this is a bug for all other field checks. Should't we change them all?

Reply:
I don't think it should be done in this scope, especially since other things, like memory, time limit and concurrency don't have such scenario because `0` is not a valid configuration and will be rejected by OW. All validation was removed from `aio cli` plugin to be performed by OW intentionally https://github.com/adobe/aio-cli-plugin-runtime/pull/317.

Reply:
our unit tests should have covered all possible input ranges, in this case it did not.